openwisp_monitoring/check/tests/test_models.py

```diff
@@ -26,6 +32,14 @@ def test_check_class(self):
         c = Check(name='Ping class check', check=self._PING)
         self.assertEqual(c.check_class, Ping)
 
+    def test_base_check_class(self):
```

Comment:
I think this test does not provide much value. I would remove it

Reply:
I had to add these tests since there were 4 uncovered lines and coverage fell by 0.1% (currently it's increased by 0.2%). If coverage is not much important then I can remove the tests though this one is verifying that you cannot derive a `Check` class from `BaseCheck` without defining `check` method (it was suggested by @nemesisdesign to raise `NotImplementedError` in all such cases. )